### PR TITLE
Product roadmap revisions from 6/13 census call

### PR DESCRIPTION
**Pull request by petermarks-gsa**
https://github.com/GSA-TTS/FAC/issues/249

### CODE REVIEW

File: docs/product/roadmap.md [hunk 1]
```diff
@@ -56,7 +57,6 @@ IMPORTANT NOTE: This system does not meet our full obligations as the Clearingho
     * Very basic, like year in excel match year submitted? UEI’s match?
 * Non-xls input to audit packages
 * Copying over FAQ’s and other pages
-* Submitting revisions to an audit package
```
Comment (MelissaBraxton) [inline docs/product/roadmap.md:59]: We'll want to make sure we reflect the need for this down the road, but maybe not for MVP.

**Review by MelissaBraxton — CHANGES_REQUESTED** [review]
Looks good, but let's also make sure we copy the revision need as part of a post MVP phase.

File: docs/product/roadmap.md (hunk 1 above)
Comment (petermarks-gsa) [reply docs/product/roadmap.md:59]: My impression from yesterday's [meeting](https://docs.google.com/document/d/1YP2K-WD1Uw1D8RvMYvxj3w57DV2uPyG4guqWu7AumLo/edit#) is that auditee/user-generated revisions is for sure not MVP given how tough, imperfect and unused solutions in this space seem to be. When we asked Mark directly what the cost of no revisions is, he responded that "For the user generated ones, Id imagine a lot aren’t material[y harmful]". I'm not sure those kind of revisions belong in the "Full FAC functionality release" either. This seems more like a wish list item we could potentially track its own section. 

That said, it sounds like we need to keep invalid resubmission flow on the roadmap and also provide a resubmission workaround for user-generated revisions. I've added that to milestone 3 as it seems to go along with the status and user business logic theme.

Comment (jadudm) [reply docs/product/roadmap.md:59]: Something Mark said in passing (I used this as an example, not realizing it was part of a current discussion), and he pointed out that once it is submitted, it starts its journey into the backend, and there's... *process* that they then invoke? Like, they message all the cognizant agencies when a grantee/auditor says they made a material error, and need to pull it back and revise?

Only adding this to the convo here to signal that it sounds like there's complexity (but, also ack'ing that y'all may have already uncovered that complexity in your research/convos).

**Review by MelissaBraxton — APPROVED** [review]
LGTM!

**Pull request merged**